packages/kit/src/runtime/server/page/fetch.js

```diff
@@ -161,10 +163,7 @@ export function create_fetch({ event, options, state, route, prerender_default }
 			// - sub.my.domain.com WILL receive cookies
 			// ports do not affect the resolution
 			// leading dot prevents mydomain.com matching domain.com
-			if (
-				`.${new URL(requested).hostname}`.endsWith(`.${event.url.hostname}`) &&
-				opts.credentials !== 'omit'
-			) {
+			if (`.$url.hostname}`.endsWith(`.${event.url.hostname}`) && opts.credentials !== 'omit') {
```

Comment:
I think there's a missing `{` - which, if I'm correct , makes we wonder how tests are still all passing
```suggestion
			if (`.${url.hostname}`.endsWith(`.${event.url.hostname}`) && opts.credentials !== 'omit') {
```

Reply:
whoops. i think this behaviour is just untested